app/src/components/widgets/charts/launchExecutionAndIssueStatistics/issueStatisticsChart.jsx

```diff
@@ -50,7 +50,7 @@ const getResult = (widget) => widget.content.result[0] || widget.content.result;
     getDefectLink: (params) => defectLinkSelector(state, params),
   }),
   {
-    redirect,
+    redirect: redirectAction,
```

Comment:
I think `redirect` will be confusing in this way. My suggestion is:
`navigate: (link) => link`

Reply:
I agree. Now it isn't actually a redirect.

Reply:
Agree. I've fixed it.